cypress/integration/mocked/change_password.spec.ts

```diff
@@ -159,14 +159,14 @@ describe('Password change flow', () => {
 
   context('Email shown on page', () => {
     it('shows the users email address on the page', () => {
-      cy.mockNext(200, fakeValidationRespone());
+      cy.mockNext(200, fakeValidationResponse());
       cy.visit(`/reset-password/fake_token`);
-      cy.contains(fakeValidationRespone().user.primaryEmailAddress);
+      cy.contains(fakeValidationResponse().user.primaryEmailAddress);
     });
   });
 
   context('Valid password entered', () => {
-    it('shows password change success screen, with a default redirect button.', () => {
+    it.only('shows password change success screen, with a default redirect button.', () => {
```

Comment:
I imagine this was just for testing?

Reply:
ah nice one, I had in a commit but forgot to push :)